parachain/config.json

```diff
@@ -22,7 +22,16 @@
 			"wsPort": 9988,
 			"port": 31200,
 			"balance": "1000000000000000000000",
-			"flags": ["--rpc-cors=all", "--enable-offchain-indexing", "--discover-local", "--", "--execution=wasm"]
+			"flags": [
+				"-lbasic_authorship=trace,import_header=trace,runtime=debug",
+				"--execution=native",
```

Comment:
@vgeddes I'm adding `--execution=native`. Without it, the PoW merkle proofs fail and Eth header imports won't work. I think it relates to the hashing functions but haven't gotten to the bottom of it yet

Reply:
ah that's concerning, but sure, fine for now :+1:

Reply:
Um what does that do? Does it mess with the WASM code/stop compilation to wasm? May be issues with connecting to the relay chain if so

Reply:
The parachain node on its host can execute either using its native or WASM runtime (both are builtin to the node executable). Note this is all after the WASM has been compiled and registered on the relay chain. Even if running natively, the node can only execute blocks that were validated on the relay chain using the registered WASM validation function.